### ISSUE TITLE
Users may want to receive the native token(DAI) on gnosis instead of WXDAI

### ISSUE THREAD

**Issue by 0xcodercrane** [ubiquity/ubiquibot#326]
Here's another thing I am thinking for the gnosis chain. so basically users want to DAI instead of WXDAI on gnosis because it is the native token. Its not a blocker but we might need to think about that on users end

_Originally posted by @0xcodercrane in https://github.com/ubiquity/ubiquibot/issues/316#issuecomment-1558558420_

**Comment by 0xcodercrane** [comment]
I want your input @pavlovcik ^^

**Comment by rndquu** [comment]
Uniswap's permit2 supports only [ERC20 compatible tokens](https://github.com/Uniswap/permit2/blob/main/src/SignatureTransfer.sol#L123)

We could fork uniswap's permit2 and add native token support (or think of another solution) but that is too much effort for a solution that we will abandon in a year when gas prices in ethereum mainnet won't be so volatile.

**Comment by pavlovcik** [comment]
Yeah let's focus on ERC20 support, and then maybe in v2 we can focus on network native gas token support. Supporting payments in ETH/xDAI could be nice after all! 

I expect that it wouldn't be too much bloat to the codebase to add a few more functions for gas token support!

**Comment by pavlovcik** [comment]
Or should I make this bounty `Time: <1 Week`?

**Comment by 0xcodercrane** [comment]
It wouldn't be too much it will just require a new method with wrap/unwrap feature.
